### PR TITLE
Replace racially charged terminology in docs

### PR DESCRIPTION
**Pull request by Dean-Coakley**
### Proposed changes
Remove all references to racially charged terminology in docs, with the exception of feature dependent references.

### Checklist
Before creating a PR, run through this checklist and mark each as complete.

- [x] I have read the [CONTRIBUTING](https://github.com/nginxinc/kubernetes-ingress/blob/master/CONTRIBUTING.md) doc
- [x] I have updated necessary documentation
- [x] I have rebased my branch onto master
- [x] I will ensure my PR is targeting the master branch and pulling from my branch from my own fork

### CODE REVIEW

File: deployments/helm-chart/README.md [hunk 1]
```diff
@@ -178,7 +178,7 @@ Parameter | Description | Default
 `controller.healthStatusURI` | Sets the URI of health status location in the default server. Requires `controller.healthStatus`. | "/nginx-health"
 `controller.nginxStatus.enable` | Enable the NGINX stub_status, or the NGINX Plus API. | true
 `controller.nginxStatus.port` | Set the port where the NGINX stub_status or the NGINX Plus API is exposed. | 8080
-`controller.nginxStatus.allowCidrs` | Whitelist IPv4 IP/CIDR blocks to allow access to NGINX stub_status or the NGINX Plus API. Separate multiple IP/CIDR by commas. | 127.0.0.1
+`controller.nginxStatus.allowCidrs` | Allowlist IPv4 IP/CIDR blocks access to NGINX stub_status or the NGINX Plus API. Separate multiple IP/CIDR by commas. | 127.0.0.1
```
Comment (jputrino) [inline deployments/helm-chart/README.md:181]:
```suggestion
`controller.nginxStatus.allowCidrs` | Add IPv4 IP/CIDR blocks to the allow list for NGINX stub_status or the NGINX Plus API. Separate multiple IP/CIDR by commas. | 127.0.0.1
```

Per Microsoft style guide (which we follow), "allow list" should be two words, used as a noun (https://docs.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/w/whitelist)

File: cmd/nginx-ingress/main.go [hunk 2]
```diff
@@ -124,7 +124,7 @@ var (
 	leaderElectionLockName = flag.String("leader-election-lock-name", "nginx-ingress-leader-election",
 		`Specifies the name of the ConfigMap, within the same namespace as the controller, used as the lock for leader election. Requires -enable-leader-election.`)
 
-	nginxStatusAllowCIDRs = flag.String("nginx-status-allow-cidrs", "127.0.0.1", `Whitelist IPv4 IP/CIDR blocks to allow access to NGINX stub_status or the NGINX Plus API. Separate multiple IP/CIDR by commas.`)
+	nginxStatusAllowCIDRs = flag.String("nginx-status-allow-cidrs", "127.0.0.1", `Allowlist IPv4 IP/CIDR blocks access to NGINX stub_status or the NGINX Plus API. Separate multiple IP/CIDR by commas.`)
```
Comment (jputrino) [inline cmd/nginx-ingress/main.go:127]:
```suggestion
	nginxStatusAllowCIDRs = flag.String("nginx-status-allow-cidrs", "127.0.0.1", `Add IPv4 IP/CIDR blocks to the allow list for NGINX stub_status or the NGINX Plus API. Separate multiple IP/CIDR by commas.`)
```

File: deployments/helm-chart/values.yaml [hunk 3]
```diff
@@ -180,7 +180,7 @@ controller:
     ## Set the port where the NGINX stub_status or the NGINX Plus API is exposed.
     port: 8080
 
-    ## Whitelist IPv4 IP/CIDR blocks to allow access to NGINX stub_status or the NGINX Plus API. Separate multiple IP/CIDR by commas.
+    ## Allowlist IPv4 IP/CIDR blocks access to NGINX stub_status or the NGINX Plus API. Separate multiple IP/CIDR by commas.
```
Comment (jputrino) [inline deployments/helm-chart/values.yaml:183]:
```suggestion
    ## Add IPv4 IP/CIDR blocks to the allow list for NGINX stub_status or the NGINX Plus API. Separate multiple IP/CIDR by commas.
```

File: docs-web/configuration/global-configuration/command-line-arguments.md [hunk 4]
```diff
@@ -125,7 +125,7 @@ Below we describe the available command-line arguments:
 
 .. option:: -nginx-status-allow-cidrs <string>
 
-	Whitelist IPv4 IP/CIDR blocks to allow access to NGINX stub_status or the NGINX Plus API.
+	Allowlist IPv4 IP/CIDR blocks access to NGINX stub_status or the NGINX Plus API.
```
Comment (jputrino) [inline docs-web/configuration/global-configuration/command-line-arguments.md:128]:
```suggestion
	Add IPv4 IP/CIDR blocks to the allow list for NGINX stub_status or the NGINX Plus API.
```

File: docs-web/installation/installation-with-helm.md [hunk 5]
```diff
@@ -263,7 +263,7 @@ The following tables lists the configurable parameters of the NGINX Ingress cont
      - Set the port where the NGINX stub_status or the NGINX Plus API is exposed.
      - 8080
    * - ``controller.nginxStatus.allowCidrs``
-     - Whitelist IPv4 IP/CIDR blocks to allow access to NGINX stub_status or the NGINX Plus API. Separate multiple IP/CIDR by commas.
+     - Allowlist IPv4 IP/CIDR blocks access to NGINX stub_status or the NGINX Plus API. Separate multiple IP/CIDR by commas.
```
Comment (jputrino) [inline docs-web/installation/installation-with-helm.md:266]:
```suggestion
     - Add IPv4 IP/CIDR blocks to the allow list for NGINX stub_status or the NGINX Plus API. Separate multiple IP/CIDR by commas.
```

**Review by jputrino — APPROVED** [review]
Should be good to go once comments are addressed. Thanks!

**Review by lucacome — APPROVED** [review]

**Review by pleshakov — APPROVED** [review]

**Pull request merged**